monix-eval/shared/src/main/scala/monix/eval/Task.scala

```diff
@@ -154,7 +161,7 @@ import scala.util.{Failure, Success, Try}
   *
   * {{{
   *   // Some array of tasks, you come up with something good :-)
-  *   val list: Seq[Task[Int]] = ???
+  *   val list: Seq[Task[Int]] = Seq(Task(10), Task(20), Task(30))
   *
   *   // Split our list in chunks of 30 items per chunk,
   *   // this being the maximum parallelism allowed
```

Comment:
Actually, I didn't realize it was splitting the list in the chunk of 30 below... Do `Seq.tabulate(100)(Task(_))`?

Reply:
looks good